vitrina/uapi/models.py

```diff
@@ -91,10 +91,14 @@ class Meta:
             )
         ]
 
-
     def save(self, *args, **kwargs) -> None:
         self.codename = self.get_codename(self.title)
 
+        if (
+            update_fields := kwargs.get("update_fields")
+        ) is not None and "title" in update_fields:
```

Comment:
```suggestion
        if update_fields := kwargs.get("update_fields", {}) and "title" in update_fields:
```

Fits into 120 char and is None safe

Reply:
fyi, ^ will need to put this part into brackets to work:
`update_fields := kwargs.get("update_fields")`

Reply:
Agree with both of you. Fixed in 797ada5082a2f6ca2ed2730bd11268a8a28e03cc